CommonMark/Formatters/HtmlFormatterSlim.cs

```diff
@@ -621,6 +628,15 @@ private static void InlinesToHtml(HtmlTextWriter writer, Inline inline, CommonMa
                         stackWithinLink = withinLink;
                         break;
 
+                    case InlineTag.Placeholder:
+                        // the slim formatter will treat placeholders like literals, without applying any further processing
```

Comment:
Yes, this is a completely valid point. I assume that it can be safely left as is - even if we figure out how to enable the slim formatter to do this, it should be easy to implement it by preserving the current usage as well.